momento-sdk/src/main/java/momento/sdk/config/Configurations.java

```diff
@@ -0,0 +1,87 @@
+package momento.sdk.config;
+
+import java.time.Duration;
+import momento.sdk.config.transport.GrpcConfiguration;
+import momento.sdk.config.transport.StaticTransportStrategy;
+import momento.sdk.config.transport.TransportStrategy;
+
+/** Prebuilt {@link Configuration}s for different environments. */
+public class Configurations {
+
+  /**
+   * Provides defaults suitable for a medium-to-high-latency dev environment. Permissive timeouts,
+   * retries, and relaxed latency and throughput targets.
+   */
+  public static class Laptop extends Configuration {
+
+    private Laptop(TransportStrategy transportStrategy) {
+      super(transportStrategy);
+    }
+
+    /**
+     * Provides the latest recommended configuration for a dev environment.
+     *
+     * <p>This configuration may change in future releases to take advantage of improvements we
+     * identify for default configurations.
+     *
+     * @return the latest Laptop configuration
+     */
+    public static Configuration Latest() {
+      final TransportStrategy transportStrategy =
+          new StaticTransportStrategy(new GrpcConfiguration(Duration.ofMillis(15000)));
```

Comment:
Can we put the harcoded millisecond values as a global variable?
LATEST_LAPTOP_MILLIS, IN_REGION_MILLIS, LOW_LATENCY_MILLIS - something along these lines. Might help we we want to change these values in future?

Reply:
Got it, thanks for clarifying!